node/node.go

```diff
@@ -563,6 +529,29 @@ func (n *Node) GetAllESDTTokens(address string) (map[string]*esdt.ESDigitalToken
 	return allESDTs, nil
 }
 
+func extractTokenIDAndNonceFromTokenKey(tokenKey []byte) ([]byte, uint64) {
```

Comment:
I think we might have a function like this somewhere.

Reply:
extracted